perllib/FixMyStreet/Reporting.pm

```diff
@@ -305,6 +306,46 @@ sub generate_csv {
 
 # Output code
 
+sub cache_dir {
+    my $self = shift;
+
+    my $cfg = FixMyStreet->config('PHOTO_STORAGE_OPTIONS');
+    my $dir = $cfg ? $cfg->{UPLOAD_DIR} : FixMyStreet->config('UPLOAD_DIR');
+    $dir = path($dir, "dashboard_csv")->absolute(FixMyStreet->path_to());
+    my $subdir = $self->user ? $self->user->id : 0;
```

Comment:
Should there not always be a user here?

Reply:
No, kiitc uses the cobrand hook to allow non-authenticated access to the CSV export.

Reply:
Though they will still be using export=1 and not be large, so wouldn't hit this. But still, might as well be safe.